fetch/programmer-humour/styles.css

```diff
@@ -0,0 +1,25 @@
+html {
+  background-color: rgb(252, 254, 211);
+}
+
+#comic-container {
```

Comment:
I don't see this styling (comic-container)being used in your application. If it's not being used, it's best to remove it. Or else, please review your index.html and make sure your div id names are correct.